Aardvark.xcodeproj/project.pbxproj

```diff
@@ -122,26 +132,50 @@
 /* End PBXFileReference section */
 
 /* Begin PBXFrameworksBuildPhase section */
-		EAD1441E19E073FB0065A1FF /* Frameworks */ = {
+		4551A22A1BD9B41000F216D0 /* Frameworks */ = {
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				EA6E219119E8682B0097BF4F /* MessageUI.framework in Frameworks */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
-		EAD1442919E073FB0065A1FF /* Frameworks */ = {
+		4551A2341BD9B41000F216D0 /* Frameworks */ = {
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				EAAB38AD19E3191D00161A54 /* libAardvark.a in Frameworks */,
```

Comment:
Projects that consume Aardvark as a submodule may still need the `.a`. We should create new targets that create the `.framework` rather than modifying the existing targets to create a `.framework` instead of a `.a`.

Here's how we did it in Valet: https://github.com/square/Valet/pull/42

Btw, thanks for taking on this task!